src/settings/GeneralForm.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import {
+  Form,
+} from 'react-final-form';
+
+import {
+  Pane,
+  PaneFooter,
+  TextField,
+  Checkbox,
+  Select,
+  TextArea,
+  Button,
+} from '@folio/stripes/components';
+
+import RowComponent from './RowComponent';
+
+const TIME_GRANULARITY_SHORT_FORMAT = 'YYYY-MM-DD';
```

Comment:
I would suggest move constants to separate file in order to have an ability reuse them in future.

Reply:
This is specific settings just for oai-pmh granularity select box values. Granularity select box values cannot be reusable in any other part of our application(UI part).